app/pages/HomePage/components/GuideList/GuideList.jsx

```diff
@@ -21,25 +17,14 @@ import Imghygiene from './assets/hygiene.jpg';
 import Imgdomesticviolence from './assets/domesticviolence.jpg';
 import Imglgbtq from './assets/lgbtq.jpg';
 
-function typeform(event, link) {
-  const typeformReference = typeformEmbed.makePopup(
-    link,
-    {
-      mode: 'popup',
-      hideFooters: true,
-    },
-  );
-  typeformReference.open();
-}
-
 const GuideCard = ({
-  img, link, name, isTypeform = false,
+  img, link, name, categoryId,
 }) => {
-  // if this is typeform, we open the typeform modal on click.
-  // Otherwise, we just attach the link as an href.
-  const anchorTagProps = isTypeform ? {
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const anchorTagProps = categoryId ? {
     role: 'button',
-    onClick: e => { typeform(e, link); },
+    onClick: () => setModalOpen(true),
```

Comment:
I still have an open question about whether this is replacing all typeform guides or all guides. The answer to that may change this a bit. https://sheltertech.slack.com/archives/C013NS0057A/p1590026656001500

Reply:
turns out that this is not replacing any of the typeform guides. Those will remain as-is and we'll be creating new categories for this new flow. I've put all the typeform links back in and removed the hardcoded categoryIds for now. I'll hook up the right categories once they are available on the backend.